client/src/pages/Home/HeroBanner.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+
+import { Button } from '@/components/ui/button';
+import { TypewriterEffect } from '@/components/ui/typewriter-effect';
+import heroImg from '../../assets/HeroImage.jpg';
+import { FaCheckCircle } from 'react-icons/fa';
+import { FaChevronRight } from 'react-icons/fa';
+
+interface chatBoxComp {
+  text: string;
+  bgColor: string;
+  textColor: string;
+  borderRadius: string;
+  top?: string;
+  bottom?: string;
+  left?: string;
+  right?: string;
+}
+
+const ChatBox: React.FC<chatBoxComp> = ({
+  text,
+  bgColor,
+  textColor,
+  borderRadius,
+  top,
+  bottom,
+  left,
+  right,
+}) => {
+  let styleAttributes: { [key: string]: string } = {};
+  if (top) styleAttributes['top'] = top;
+  if (bottom) styleAttributes['bottom'] = bottom;
+  if (left) styleAttributes['left'] = left;
+  if (right) styleAttributes['right'] = right;
+
+  styleAttributes['backgroundColor'] = bgColor;
+  styleAttributes['color'] = textColor;
+  styleAttributes['border-radius'] = borderRadius;
+  console.log(styleAttributes);
+
+  return (
+    <div
+      className="rounded-lg p-4 absolute drop-shadow-xl"
+      style={styleAttributes}
+    >
+      {text}
+    </div>
+  );
+};
+
+const HeroBanner = () => {
```

Comment:
Single File Single Responsibility
Also try to separate logic from rendering to adhere to coding practices.